SetK.md

```diff
@@ -0,0 +1,149 @@
+autoscale: true
+footer:  [@calvellido](https://twitter.com/calvellido) [@47deg](https://twitter.com/47deg) :: [Λrrow](http://arrow-kt.io) :: [http://arrow-kt.io/docs/datatypes/setK/](http://arrow-kt.io/docs/datatypes/setK/)
+slidenumbers: true
+
+# SetK
+
+__`SetK`__ is a data type used in __Λrrow__ as a higher kinded wrapper around the Kotlin Set collection interface.
+
+---
+
+# SetK :: Constructor
+
+__`SetK`__ can be created by directly passing a __Set__.
+
+```kotlin
+SetK(setOf(1, 2, 5, 3, 2))
+// SetK(set=[1, 2, 5, 3])
+```
+
+---
+
+# SetK :: k()
+
+It can also be created invoking the __`k()`__ function on a __Set__.
+
+```kotlin
+setOf(1, 2, 5, 3, 2).k()
+// SetK(set=[1, 2, 5, 3])
+```
+
+---
+
+# SetK :: pure()
+
+Or through the __`pure()`__ function on __SetK__.
+
+```kotlin
+SetK.pure(1)
+// SetK(set=[1])
+```
+
+---
+
+# SetK :: Instances
```

Comment:
We should mention here that Set does not derive `Functor`, `Applicative`, `Monad` and `Traverse` because of the lack of ordering guarantees on `Set`